data/mods/Dark-Skies-Above/harvest.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "id": "dks_alien_cyborg",
+    "id": "dkz_alien_cyborg",
```

Comment:
Why is every ID changed from "dks" to "dkz"?

This doesn't make any sense, and not only that, it breaks saves due to no migration.